src/app/layout.tsx

```diff
@@ -65,21 +85,24 @@ export default function RootLayout({ children }: RootLayoutProps) {
       </div>
     </div>
   );
+}
 
-  const rightSideContent = (
-    <div
+function RightSideContent() {
+  return (
+    <a
       data-state="Default"
       className="px-4 py-3 bg-gray-900 rounded shadow-[-3px_4px_13.899999618530273px_0px_rgba(217,70,239,0.50)] outline outline-offset-[-1px] outline-fuchsia-500 flex justify-center items-center overflow-hidden"
+      href="https://ory.sh"
```

Comment:
maybe we could link https://www.ory.sh/docs/kratos/social-signin/fedcm?

@jhickmanit @pi1814 wdyt?

Reply:
Yeah, that works.